cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -179,6 +179,8 @@ func CreateGceManager(configReader io.Reader, discoveryOpts cloudprovider.NodeGr
 		if err := manager.cache.RegenerateInstancesCache(); err != nil {
 			klog.Errorf("Error while regenerating Mig cache: %v", err)
 		}
+		klog.Info("Invalidating MIG basename cache")
+		manager.cache.InvalidateAllMigBasenames()
```

Comment:
Do we need this call? I realize it was previously filled in regenerateInstancesCache, but:
- does basename ever change? 
- if it does/we care, does it matter that there's a race condition (another goroutine may access cache after regenerating instances, but before invalidating this)?

Reply:
> Do we need this call? I realize it was previously filled in regenerateInstancesCache, but:

Not strictly. It should not change. This is just a "just-in-case-our-cache-got-rotten" kinda call.
I can drop it if you prefer.
 
> * does basename ever change?

I do not believe so.

> * if it does/we care, does it matter that there's a race condition (another goroutine may access cache after regenerating instances, but before invalidating this)?

Reply:
I will drop this.